draft-ietf-rats-architecture.md

```diff
@@ -1719,28 +1731,31 @@ as discussed in {{epochfreshness}}, and the message is received during
 that window, the Attestation Result is accepted as fresh, and otherwise
 it is rejected as stale.
 
-~~~~
+~~~~ aasvg
                   .-------------.
    .----------.   | Epoch ID    |   .----------.  .---------------.
    | Attester |   | Distributor |   | Verifier |  | Relying Party |
-   '----------'   '-------------'   '----------'  '---------------'
+   '----+-----'   '------+------'   '-----+----'  '-------+-------'
+        |                |                |               |
      time(VG_a)          |                |               |
         |                |                |               |
         ~                ~                ~               ~
         |                |                |               |
-     time(IR_a)<------I--+--I--------time(IR_v)----->time(IR_r)
+     time(IR_a) <-----I--o--I------> time(IR_v) ---> time(IR_r)
         |                |                |               |
      time(EG_a)          |                |               |
-        |---Evidence--------------------->|               |
+        |                |                |               |
+        +---Evidence--------------------->|               |
         |   {I,time(EG_a)-time(VG_a)}     |               |
         |                |                |               |
         |                |           time(RG_v)           |
-        |<--Attestation Result------------|               |
+        |                |                |               |
+        |<--Attestation Result------------+               |
         |   {I,time(RX_v)-time(RG_v)}     |               |
         |                |                |               |
-     time(IR'_a)<-----I'-+--I'-------time(IR'_v)---->time(IR'_r)
+     time(IR'_a) <----I'-o--I'-----> time(IR'_v) --> time(IR'_r)
```

Comment:
In the rendering that I'm seeing on my desktop, I see: 
  http://www.sandelman.ca/tmp/mcrcapture/2940.2022-06-13/capture1.png
note how I' to the left, but not I' to the right.  Bug in aasvg?